src/components/ComputeModal.js

```diff
@@ -1051,7 +1065,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
             value: computeConfig.autopauseThreshold,
             hidden: !isAutopauseEnabled(computeConfig.autopauseThreshold),
             tooltip: !isAutopauseEnabled(computeConfig.autopauseThreshold) ? 'Autopause must be enabled to configure pause time.' : undefined,
-            onChange: updateComputeConfig('autopauseThreshold')
+            onChange: () => updateComputeConfig('autopauseThreshold')
```

Comment:
I know we talked offline but submitting this review just so the PR triage automation doesn't consider the PR not-reviewed :)
```suggestion
            onChange: updateComputeConfig('autopauseThreshold')
```